nix/shell.nix

```diff
@@ -1,37 +1,11 @@
-{
-  /* This should match Nixpkgs commit in status-mobile. */
-  source ? builtins.fetchTarball {
-    url = "https://github.com/NixOS/nixpkgs/archive/e7603eba51f2c7820c0a182c6bbb351181caa8e7.tar.gz";
-    sha256 = "sha256:0mwck8jyr74wh1b7g6nac1mxy6a0rkppz8n12andsffybsipz5jw";
-  },
-  pkgs ? import (source){
-    config = {
-      allowUnfree = true;
-      android_sdk.accept_license = true;
-    };
-    overlays = [
-      (self: super: {
-        androidPkgs = pkgs.androidenv.composeAndroidPackages {
-          toolsVersion = "26.1.1";
-          platformToolsVersion = "33.0.3";
-          buildToolsVersions = [ "31.0.0" ];
-          platformVersions = [ "31" ];
-          cmakeVersions = [ "3.18.1" ];
-          ndkVersion = "22.1.7171670";
-          includeNDK = true;
-          includeExtras = [
-            "extras;android;m2repository"
-            "extras;google;m2repository"
-          ];
-        };
-      })
-    ];
-  }
-}:
+{ config ? {}
+, pkgs ? import ./pkgs.nix { inherit config; } }:
```

Comment:
Since you are moving this file here you will need to import it at the root level of the repo using `shell.nix`.

Also, the move of `nix.conf` will also require changes in the `Makefile` to make work. And it need to be updated too.